e2e/testgames/game/Dockerfile

```
@@ -17,12 +17,36 @@
 
 
 ################################
-# Runtime Image
+# Runtime Image - Normal
 ################################
 FROM gcr.io/distroless/base-debian12 AS runtime
```

Hadolint

[warning] Always tag the version of an image explicitly

(DL3006)